src/h5web/providers/hsds/models.ts

```diff
@@ -26,9 +26,11 @@ export interface HsdsDatatypeResponse {
 }
 
 export interface HsdsAttributesResponse {
-  attributes: HDF5Attribute[];
+  attributes: HsdsAttributesList;
 }
 
+export type HsdsAttributesList = Omit<HDF5Attribute, 'value'>[];
+
```

Comment:
The introduction of this new type is perhaps not necessary and/or badly named.

Reply:
Pas besoin d'un sous-type du tout ;)